### PR TITLE
Fix color serialization

### PR DESCRIPTION
**Pull request by willsb**
Squash whenever

### CODE REVIEW

**Comment by willsb** [comment]
I thought about Foundation because that's the only place where we can manually create a project, but you are correct

File: Toggl.Ultrawave.Tests/Models/ProjectTests.cs [hunk 1]
```diff
@@ -45,5 +46,15 @@ public void CanBeSerialized()
...
         }
+
+        [Fact]
+        public void ColorIsAlwaysLowercaseWhenSerialized()
+        {
+            var serializer = new Ultrawave.Serialization.JsonSerializer();
+            var json = serializer.Serialize(validProject);
+            var newProject = serializer.Deserialize<Project>(json);
+
+            newProject.Color.Should().Be(newProject.Color.ToLower());
```
Comment (simonrozsival) [inline Toggl.Ultrawave.Tests/Models/ProjectTests.cs]: This test is not correct - you test that the color value is lowercase after _deserialization_, not after serialization. Use regexp instead:
```csharp
json.Should().MatchRegex("^.*\"color\":\"#f61d38\".*$");
json.Should().NotMatchRegex("^.*\"color\":\"#F61D38\".*$");
```

File: Toggl.Ultrawave.Tests/Models/ProjectTests.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using FluentAssertions;
+using Newtonsoft.Json;
```
Comment (simonrozsival) [inline Toggl.Ultrawave.Tests/Models/ProjectTests.cs]: 🔥

File: Toggl.Ultrawave/Models/Project.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using Newtonsoft.Json;
```
Comment (simonrozsival) [inline Toggl.Ultrawave/Models/Project.cs]: 🔥

**Pull request merged**